Port to 26.2 (#645)
FabricMC/fabric-docs: develop/items/custom-tools.md

## Registering Tool Items {#registering-tool-items}

> Using the same utility functions as in the [Creating Your First Item](./first-item) guide, you can create your tool items:

Reviewer: ```suggestion
You can create your tool items by using the same classes and functions that we implemented in the [Creating Your First Item](./first-item) guide:
```
